Just for completeness' sake, here's the error I see after running `yarn install` and `yarn dev` on the repo's `main` branch:

```
❯ yarn dev
yarn run v1.22.19
$ astro dev
10:00:06 AM [vite] Error when evaluating SSR module /Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/astro.config.mjs: failed to import "@astrojs/tailwind"
|- Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'tailwindcss' imported from /Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/node_modules/@astrojs/tailwind/dist/index.js
    at new NodeError (node:internal/errors:405:5)
    at packageResolve (node:internal/modules/esm/resolve:887:9)
    at moduleResolve (node:internal/modules/esm/resolve:936:20)
    at defaultResolve (node:internal/modules/esm/resolve:1129:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36)

[astro] Unable to load your Astro config

node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Failed to load PostCSS config: Failed to load PostCSS config (searchPath: /Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket): [Error] Loading PostCSS Plugin failed: Cannot find module 'tailwindcss'
Require stack:
- /Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/postcss.config.cjs

(@/Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/postcss.config.cjs)
Error: Loading PostCSS Plugin failed: Cannot find module 'tailwindcss'
Require stack:
- /Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/postcss.config.cjs

(@/Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/postcss.config.cjs)
    at load (file:///Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:36429:11)
    at file:///Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:36454:16
    at Array.map (<anonymous>)
    at plugins (file:///Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:36453:8)
    at processResult (file:///Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:36523:14)
    at file:///Users/mitch/Dev/reference-only/astro-blog-template-personal-eval/tiny-rocket/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:36647:14]

Node.js v18.17.1
error Command failed with exit code 1.
```

See my comment below, but it appears that `npm` is installing `tailwindcss` as a peer dep from `@astrojs/tailwind@5.0.2` where `yarn` is not. I don't know why yet.

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
     "dependencies": {
         "@astrojs/rss": "^3.0.0",
         "@tailwindcss/typography": "^0.5.10",
-        "preact": "^10.15.1"
+        "preact": "^10.15.1",
+        "tailwindcss": "^3.3.3"
     }
 }

